lib/merukari/merukari_screen.dart

```diff
@@ -0,0 +1,522 @@
+import 'package:flutter/material.dart';
+
+class MerukariScreen extends StatelessWidget {
+  const MerukariScreen({Key? key}) : super(key: key);
+  static const lightGray = Color.fromRGBO(231, 231, 231, 1);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: _buildAppBar(),
+      body: _buildBody(context),
+      bottomNavigationBar: _buildBottomNavigationBar(),
+      floatingActionButton: SizedBox(
+        width: 60,
+        height: 60,
+        child: FloatingActionButton.extended(
+          elevation: 5,
+          shape: const CircleBorder(),
+          backgroundColor: const Color.fromRGBO(195, 90, 75, 1),
+          onPressed: () {},
+          label: SizedBox(
+            width: 100,
+            height: 50,
+            child: Column(
+              children: const [
+                Padding(
+                  padding: EdgeInsets.only(top: 5.0),
+                  child: Icon(
+                    Icons.photo_camera,
+                    size: 23,
+                  ),
+                ),
+                Text(
+                  '出品',
+                  style: TextStyle(fontSize: 11),
+                ),
+              ],
+            ),
+          ), //テキスト//アイコン
+        ),
+      ),
+    );
+  }
+
+  AppBar _buildAppBar() {
+    return AppBar(
+      elevation: 1,
+      backgroundColor: Colors.white,
+      automaticallyImplyLeading: false,
+      title: Row(
+        mainAxisAlignment: MainAxisAlignment.center,
+        mainAxisSize: MainAxisSize.min,
+        children: const [
+          Text(
+            "出品",
+            style: TextStyle(
+                fontSize: 20, color: Colors.black, fontWeight: FontWeight.bold),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildBody(context) {
+    return SingleChildScrollView(
+      child: Container(
+        color: const Color.fromRGBO(239, 239, 239, 1),
+        child: Column(
+          children: [
+            Padding(
+              padding: const EdgeInsets.all(20.0),
+              child: ClipRRect(
+                borderRadius: BorderRadius.circular(7),
+                child: Image.asset('images/メルカリ_トップ画.png'),
+              ),
+            ),
+            Padding(
+              padding: const EdgeInsets.only(left: 20.0),
+              child: Row(
+                children: const [
+                  Text(
+                    '出品へのショートカット',
+                    style: TextStyle(fontWeight: FontWeight.bold),
+                  ),
+                ],
+              ),
+            ),
+            Padding(
+              padding: const EdgeInsets.only(bottom: 20, top: 20),
```

Comment:
topとbottomで同じ値の場合には以下のようすると良いかと思います！
```suggestion
            Padding(
              padding: const EdgeInsets.symmetric(vertical: 20),
```

Reply:
了解です！ありがとうございます！